### DISCUSSION TITLE
breast_density_challenge  accuracy

### DISCUSSION THREAD

**Discussion by nickersonff** [Project-MONAI/tutorials#1519] [Q&A]
Hello everyone,

I'm currently attempting to run the breast_density_challenge code, but I'm encountering a significant issue with my local accuracy – it's quite low (about 0.5). I would greatly appreciate any guidance or suggestions on how to improve this accuracy. Thank you in advance for your help!

**Comment by KumoLiu** [comment]
Hi @nickersonff, this may differ from task to task; perhaps you can simply see if the results improve by modifying the hyperparameters such as learning rate, optimizer, et. al. Otherwise you may check your input and output to identify the issue.

Thanks!
